src/components/Explore/Dropdown/MenuButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import {
+  MainButton,
+  MenuLabel,
+  ItemLabel,
+  ButtonContent,
+  Column,
+} from './Dropdown.style';
+
+const MenuButton: React.FC<{
+  onClick: any;
+  menuLabel: string;
+  buttonLabel: string;
+  open?: boolean;
+  ariaControlsId: string;
+  maxWidth?: number;
+}> = ({ onClick, open, menuLabel, buttonLabel, ariaControlsId, maxWidth }) => {
+  return (
+    <MainButton
+      aria-controls={ariaControlsId}
+      aria-haspopup="true"
+      disableRipple
+      onClick={onClick}
```

Comment:
`disableRipple` can be disabled in the `Dropdown.style` file since is being customized already